src/main.rs

```diff
@@ -83,22 +89,41 @@ fn first_encoder(connector_info: &ConnectorInfo) -> Option<EncoderHandle> {
     connector_info.encoders().iter().next().copied()
 }
 
+const DEFAULT_CARD_PATH: &str = "/dev/dri/card0";
+
+#[derive(Parser)]
+struct Args {
+    /// Device file representing the GPU
+    #[arg(long, default_value = DEFAULT_CARD_PATH)]
+    card_path: String,
```

Comment:
I wanted to suggest this to be a `PathBuf` instead but `Card::open()` takes a &str 🥲

Reply:
Could do `PathBuf::to_str().expect()` if we feel changing it to `PathBuf` would be a net benefit.

Reply:
I think we can (and should) change it to `PathBuf`;`Card::open()` is also controlled by us, `drm` crate only relies on file descriptors, not strings. Let me try.

Reply:
Dang, I completely overlooked that `Card` is ours 🙈

Thanks. Resolving!